packages/client/src/modules/post/containers/Post.jsx

```diff
@@ -9,6 +9,8 @@ import POSTS_QUERY from '../graphql/PostsQuery.graphql';
 import POSTS_SUBSCRIPTION from '../graphql/PostsSubscription.graphql';
 import DELETE_POST from '../graphql/DeletePost.graphql';
 
+const limit = 10;
```

Comment:
Minor thing: constants should be all uppercase. Another consideration is to maybe move settings like these into `config/pagination.js`